src/style.css

```diff
@@ -0,0 +1,243 @@
+/* Main Styling */
+main {
+  /* max-width: 800px; */
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+  margin: auto;
+}
+
+.container {
+  margin: auto;
+}
+
+.meals {
+  display: grid;
+
+  /* grid-template-columns: repeat(auto-fit, minmax(300px, 1fr)); */
+  grid-template-columns: repeat(1, minmax(0, 1fr));
+  gap: 1.25rem /* 20px */;
+  margin-top: 1.25rem; /* 20px */
+  padding: 20px 10px;
+}
```

Comment:
- [ ] Please remove commented codes from your project. This would make your code cleaner.